src/audio/copier/copier_dai.c

```diff
@@ -265,7 +265,7 @@ int copier_dai_create(struct comp_dev *dev, struct copier_data *cd,
 
 	/* create multi_endpoint_buffer for ALH multi-gateway case */
 	if (dai_count > 1) {
-		ret = create_endpoint_buffer(dev, cd, config, copier, type, true, 0);
+		ret = create_endpoint_buffer(dev, cd, copier, true);
```

Comment:
please drop this commit. It will be used by https://github.com/thesofproject/sof/pull/7959/commits/78a8eec9cb9a010065c3bd7294f699986fbe7015